composer.json

```diff
@@ -30,13 +30,49 @@
         }
     },
     "extra": {
-        "class": "ComposerGpgVerify\\Verifier",
+        "class": "ComposerGpgVerify\\Verify",
         "branch-alias": {
             "dev-master": "2.0.x-dev"
+        },
+        "composer-gpg-verify": {
+            "allow-unsigned": [
+                "composer/ca-bundle",
+                "composer/composer",
+                "composer/semver",
+                "composer/spdx-licenses",
+                "psr/log",
+                "justinrainbow/json-schema",
+                "seld/cli-prompt",
+                "seld/jsonlint",
+                "seld/phar-utils",
+                "symfony/console",
+                "symfony/debug",
+                "symfony/filesystem",
+                "symfony/finder",
+                "symfony/polyfill-mbstring",
+                "symfony/process"
+            ],
+            "allow-untrusted": [
```

Comment:
Being a trusted package require being a signed one first. So I think you should not require duplicating the whole list of unsigned packages in the untrusted list (it makes it much harder to identify packages which are only in the untrusted list)